File: 2.Palindrome-Number/java/Palindrome_Number.java
import java.util.HashMap;
import java.util.Map;

class Palindronme_Number {

    // other solution
    static public boolean isPalindrome(int x) {
        if (x < 0)
            return false;
        String tmp = String.valueOf(x);
        int right = tmp.length() - 1, left = 0;

        while (left < right) {
            if (tmp.charAt(left) != tmp.charAt(right))
                return false;

            left++;
            right--;
        }

        return true;
    }

    public static void main(String[] args) {
        System.out.println(isPalindrome(121));

    }
}

File: Easy/2.Palindrome-Number/javascript/Palindrome-Number.js
var isPalindrome = function (x) {
  if (x < 0) return false
  let tmp = String(x)
  for (let i = 0; i < tmp.length; i++) {
    if (tmp[i] != tmp[tmp.length - 1 - i]) return false
  }

  return true
}

console.log(isPalindrome(9))
